actionmailbox/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Add Amazon SES/SNS ingress.
```

Comment:
Worth fleshing this out with a brief sketch of the SES, S3, and SNS orchestration involved.